docs/zh/openapi-fetch/testing.md

```diff
@@ -31,37 +33,58 @@ test("my request", async () => {
 });
 ```
 
-## 模拟响应
+:::
+
+## Mocking Responses
```

Comment:
@phk422 Just a heads up: this is one section that I wanted to exist in the Chinese documentation since this is the current recommendation. Copied the English version.

No pressure whatsoever to translate it, but just wanted your feedback: is having untranslated text in localized versions better if it’s more up-to-date information?

Or is it better to remove the section?

Would love advice going forward on the best approach (for other contributors)

Reply:
@drwpow I apologize for the delayed response.

Regarding whether to keep untranslated text in the localized versions, here are my suggestions:

Do not translate specific terms: For some specialized terms, it might be more accurate and easier to understand if they are kept in English in the Chinese documentation. For example, 'Mocking Responses' can be left untranslated.

Keep the untranslated English content: If this is the latest recommendation, temporarily keeping the English text can ensure that users have access to the most up-to-date guidance. However, you can add a note in the document indicating that this section is in English and will be translated in the next update. Once you update the document, you can @ me in the PR, and I will translate it as soon as possible.

Hope this helps, and I look forward to hearing from you.